src/test/resources/application.yml

```
@@ -26,3 +26,6 @@ app:
 oauth:
   apple:
     public-key-url: https://appleid.apple.com/auth
+    client_id: ${APPLE_CLIENT_ID}
+    key_id: ${APPLE_KEY_ID}
+    team_id: ${APPLE_TEAM_ID}
```

테스트할때는 환경 변수 파일을 넣지 않아서, application.yml에 테스트용 가짜 정보들을 적어주세요

```suggestion
    client_id: test-client-id
    key_id: test-key-id
    team_id: test-team-id
```